firefox_puppeteer/api/appinfo.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class AppInfo(BaseLib):
+
+    @property
+    def browserTabsRemoteAutostart(self):
```

Comment:
Ok, but then please no camel-case but underscores. I wonder if we can return the whole appInfo object inside the execute_script() block so that we do not need dozen of helper properties like that.